app/environments/brassbirmingham/brassbirmingham/envs/consts.py

```diff
@@ -0,0 +1,520 @@
+    IndustryBuilding(BuildingName.oil, 1, 4, 5, 1, 0, 3, 3, 1, onlyPhaseOne=True),
+    IndustryBuilding(BuildingName.oil, 2, 4, 7, 1, 0, 5, 3, 1),
+    IndustryBuilding(BuildingName.oil, 3, 5, 9, 1, 0, 7, 2, 1),
+    IndustryBuilding(BuildingName.oil, 4, 6, 12, 1, 0, 9, 1, 1),
+    IndustryBuilding(BuildingName.beer, 1, 1, 5, 0, 1, 4, 4, 2),
+    IndustryBuilding(BuildingName.beer, 1, 1, 5, 0, 1, 4, 4, 2),
+    IndustryBuilding(
+        BuildingName.beer, 2, 1, 7, 0, 1, 5, 5, 2
+    ),  # add logic somewhere to add +1 beer to tier ^2 in second phase
+    IndustryBuilding(BuildingName.beer, 2, 1, 7, 0, 1, 5, 5, 2),
+    IndustryBuilding(BuildingName.beer, 3, 1, 9, 0, 1, 7, 5, 2),
+    IndustryBuilding(BuildingName.beer, 3, 1, 9, 0, 1, 7, 5, 2),
+    IndustryBuilding(BuildingName.beer, 4, 1, 9, 0, 1, 10, 5, 2),
+    IndustryBuilding(BuildingName.coal, 1, 2, 5, 0, 0, 1, 4, 2, onlyPhaseOne=True),
+    IndustryBuilding(BuildingName.coal, 2, 3, 7, 0, 0, 2, 7, 1),
+    IndustryBuilding(BuildingName.coal, 2, 3, 7, 0, 0, 2, 7, 1),
+    IndustryBuilding(BuildingName.coal, 3, 4, 8, 0, 1, 3, 6, 1),
+    IndustryBuilding(BuildingName.coal, 3, 4, 8, 0, 1, 3, 6, 1),
+    IndustryBuilding(BuildingName.coal, 4, 5, 10, 0, 1, 4, 5, 1),
+    IndustryBuilding(BuildingName.coal, 4, 5, 10, 0, 1, 4, 5, 1),
+]
+
+"""
+Starting deck
+
+Key is (str) amount of players playing
+"""
+STARTING_CARDS: Dict[str, List[Card]] = {
+    "2": [
+        LocationCard(STAFFORD),
+        LocationCard(STAFFORD),
+        LocationCard(BURTON_UPON_TRENT),
+        LocationCard(BURTON_UPON_TRENT),
+        LocationCard(CANNOCK),
+        LocationCard(CANNOCK),
+        LocationCard(TAMWORTH),
+        LocationCard(WALSALL),
+        LocationCard(COALBROOKDALE),
+        LocationCard(COALBROOKDALE),
+        LocationCard(COALBROOKDALE),
+        LocationCard(DUDLEY),
+        LocationCard(DUDLEY),
+        LocationCard(KIDDERMINSTER),
+        LocationCard(KIDDERMINSTER),
+        LocationCard(WOLVERHAMPTON),
+        LocationCard(WOLVERHAMPTON),
+        LocationCard(WORCESTER),
+        LocationCard(WORCESTER),
+        LocationCard(BIRMINGHAM),
+        LocationCard(BIRMINGHAM),
+        LocationCard(BIRMINGHAM),
+        LocationCard(COVENTRY),
+        LocationCard(COVENTRY),
+        LocationCard(COVENTRY),
+        LocationCard(NUNEATON),
+        LocationCard(REDDITCH),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.iron_works),
+        IndustryCard(CardName.coal_mine),
+        IndustryCard(CardName.coal_mine),
+        IndustryCard(CardName.pottery),
+        IndustryCard(CardName.pottery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+        IndustryCard(CardName.brewery),
+    ],
+    "3": [
```

Comment:
Wouldn't it be easier just to have these be incremental? Basically "3" only includes the additional cards to add on top of the "2" player cards? That way we don't need duplicates. 

As far as I know, there's no case where you'd play with a card with 2 players but not play with it in 3 or 4 players

Reply:
Agreed, but I'd argue it's not a total priority right now, we can always refactor this down the road

Reply:
Fair enough